packages/commons/src/utils/getAll.ts

```diff
@@ -0,0 +1,16 @@
+import { ListResult } from "pagopa-interop-models";
+
+export async function getAllFromPaginated<A>(
+  getPaginatedCall: (offset: number, limit: number) => Promise<ListResult<A>>
+): Promise<A[]> {
+  const getAllFromOffset = async (offset: number): Promise<A[]> => {
+    const limit = 50;
+    const { results } = await getPaginatedCall(offset, limit);
+
+    return results.length < limit
+      ? results
+      : results.concat(await getAllFromOffset(offset + limit));
+  };
+
+  return await getAllFromOffset(0);
+}
```

Comment:
currently, I have cherry-picked this function from an Eric PR https://github.com/pagopa/interop-be-monorepo/pull/764/files#diff-8e30f6ae0147b4b664468b1e29476bd48cb49a7928b5ed7bdf2fd5ac9653abdb.

So to avoid unnecessary conflict I prefer not to change the function naming.